### PR TITLE
Add rel noopener noreferrer to target _blank links

### PR DESCRIPTION
**Pull request by anselmbradford**
## Changes

- Adds `rel="nopener noreferrer"` to two PDF links that were missing that with `target="_blank"` (reported by @lfatty).
- Normalizes quotes to double quotes.
- Removes some extraneous whitespace.

## Testing

- Follow project installation instructions https://github.com/cfpb/college-costs#installation

### CODE REVIEW

File: paying_for_college/templates/repay_student_debt.html [hunk 1]
```diff
@@ -1110,7 +1110,7 @@ <h4 class="sample-text-header"><span class="cf-icon cf-icon-email"></span> Sampl
         <div class="content-l_col content-l_col-1-4 tip-section">
           <div class="tip_bar"></div><div class="tip_line"></div>
           <h3>TIP</h3>
-          <p>Remember, if you’re having a problem with a student loan , you can submit a complaint <a href="http://www.consumerfinance.gov/complaint/" class="internal-link" target="_blank">online</a> or call us at (855) 411-2372.</p>
+          <p>Remember, if you’re having a problem with a student loan , you can submit a complaint <a href="http://www.consumerfinance.gov/complaint/" class="internal-link" target="_blank" rel="noopener noreferrer">online</a> or call us at (855) 411-2372.</p>
```
Comment (chosak) [inline paying_for_college/templates/repay_student_debt.html]: Minor fix: remove extra space in "student loan , you can submit".

**Review by marteki — APPROVED** [review]
We've been [trying to get the proper attributes put on the target="_blank" links for a long time](https://github.com/cfpb/college-costs/pull/187#discussion-diff-67728602). Clearly, we missed some of them somehow.

There are changes made to files in this repo that aren't being pushed to the live site, but I think that's fine in order to show what we _should_ be doing everywhere for links in new tabs.

**Review by chosak — COMMENTED** [review]
This LGTM.

Do we want to also add `noreferrer` to the links generated by JavaScript, like [here](https://github.com/cfpb/college-costs/blob/master/src/disclosures/js/views/links-view.js#L41) and [here](https://github.com/cfpb/college-costs/blob/master/src/disclosures/js/views/links-view.js#L61) and [here](https://github.com/cfpb/college-costs/blob/master/src/disclosures/js/views/links-view.js#L80) and [here](https://github.com/cfpb/college-costs/blob/master/src/disclosures/js/views/links-view.js#L99)?

Also, I notice [this comment](https://github.com/cfpb/college-costs/blob/ff622ff878f565e5cbc98671d6de5bacd279e39b/paying_for_college/local_static/paying_for_college/disclosures/worksheet-dev.html#L1) -- and I wasn't able to get this file working locally -- but does it make sense to update the links in it, like [here](https://github.com/cfpb/college-costs/blob/ff622ff878f565e5cbc98671d6de5bacd279e39b/paying_for_college/local_static/paying_for_college/disclosures/worksheet-dev.html#L196)?

File: paying_for_college/templates/repay_student_debt.html (hunk 1 above)
Comment (marteki) [reply paying_for_college/templates/repay_student_debt.html]: If we make that fix here, we should also actually fix it in `django-college-costs-comparison`. This file is never deployed to the website from this repo, but comes from the other repo.

**Review by higs4281 — APPROVED** [review]

File: paying_for_college/templates/repay_student_debt.html (hunk 1 above)
Comment (anselmbradford) [reply paying_for_college/templates/repay_student_debt.html]: I fixed this here at least

**Comment by anselmbradford** [comment]
> This LGTM.
  Do we want to also add noreferrer to the links generated by JavaScript, like here and here and here and here? 
  Also, I notice this comment -- and I wasn't able to get this file working locally -- but does it make sense to update the links in it, like here?

Updated all of these. THanks!

**Pull request merged**